Makefile

```diff
@@ -65,6 +65,12 @@ fmt: .make/fmt ## Format python code with black
 blackcheck: $(VIRTUALENV) $(PYTHONFILES) ## Check if everything is formatted correctly
 	poetry run black $(BLACK_FLAGS) --check website
 
+.make/pylint: .make $(VIRTUALENV) $(PYTHONFILES)
+	cd website && DJANGO_SETTINGS_MODULE=thaliawebsite.settings poetry run pylint --load-plugins pylint_django ./**/*.py
```

Comment:
Could you try to get that working maybe? I had significant problems when trying. Especially since the website folder is not a module, it is only the root folder. So when you run Python from the root directory all the imports are no longer valid.

Reply:
Okay somehow it works like this, and it is now finding more somehow

Reply:
Changed